tests/fixtures/numeric_arrays/means-grouped-by-single-categorical.json

```diff
@@ -0,0 +1,217 @@
+                "filtered": {
+                    "unweighted": {
+                        "missing": 0,
+                        "other": 0,
+                        "selected": 6
+                    },
+                    "weighted": {
+                        "missing": 0,
+                        "other": 0,
+                        "selected": 6
+                    }
+                },
+                "filtered_complete": {
+                    "unweighted": {
+                        "missing": 0,
+                        "other": 0,
+                        "selected": 6
+                    },
+                    "weighted": {
+                        "missing": 0,
+                        "other": 0,
+                        "selected": 6
+                    }
+                }
+            },
+            "filtered": {
+                "unweighted_n": 6,
+                "weighted_n": 6
+            },
+            "measures": {
+                "count": {
+                    "data": [
+                        1,
+                        1,
+                        1,
+                        1,
+                        1,
+                        1,
+                        0
+                    ],
+                    "metadata": {
+                        "derived": true,
+                        "references": {},
+                        "type": {
+                            "class": "numeric",
+                            "integer": true,
+                            "missing_reasons": {
+                                "No Data": -1
+                            },
+                            "missing_rules": {}
+                        }
+                    },
+                    "n_missing": 1
+                },
+                "mean": {
+                    "data": [
+                        [
+                            0.0,
+                            0.0
+                        ],
+                        [
+                            1.0,
+                            10.0
+                        ],
+                        [
+                            2.0,
+                            20.0
+                        ],
+                        [
+                            3.0,
+                            30.0
+                        ],
+                        [
+                            4.0,
+                            40.0
+                        ],
+                        [
+                            5.0,
+                            50.0
+                        ],
+                        [
+                            {
+                                "?": -8
+                            },
+                            {
+                                "?": -8
+                            }
+                        ]
+                    ],
+                    "metadata": {
+                        "derived": true,
+                        "references": {},
```

Comment:
Hey, we *desperately* need the references to be passed along here. In the “movie ratings” example — we know the ratings, but now we have no idea what the movies were! Or even the variable name to know that it’s movies and not magazines! Argh!

Reply:
We have the `"subvariables"` a couple of lines below. Is that not enough?